### ISSUE TITLE
Interactive examples for Go 101 / Zig 101

### ISSUE THREAD

**Issue by nalgeon** [go101/go101#268]
Hi Tapir! Thank you for your great work on Go 101 / Generics /  Optimizations projects!

Would you be interested in making some of the code examples interactive for better understanding and engagement? (or maybe try it with your newer Zig 101 project)

Here is what these interactive examples might look like: [Go 1.22: Interactive release notes](https://antonz.org/go-1-22/)

If you're interested, I'm willing to help.

**Comment by go101** [comment]
@nalgeon 
Sorry, I just noticed this issued.
Yes, there is a plan to support interactive code examples in Go 101 book.
My current plan is to only support this for local Go101 server (by running the `go101` command).

As mentioned in many [other places](https://github.com/go101/go101/pull/261), Go 101 books will only link to Go official website and other Go 101 pages.
I have noticed that you developed codapi, which is a great service in any metric.
But I'm sorry that I have to say, at present, Go 101 will not connect to external services.

Maybe I will change my idea later. So let's keep this issue open now. :)
